tools/scripts/brokenLinkChecker/brokenLinkChecker.js

```diff
@@ -3,10 +3,13 @@
 const blc = require('broken-link-checker');
 const https = require('https');
 const url = require('url');
+const YAML = require('yamljs');
 
 const whitelist = require('./whitelist.json').ignore;
 
-const slackUrl = "https://hooks.slack.com/services/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";
```

Comment:
I will disable this one after after merging.  We fortunately already had another one for #infra-staging which happens to already show up in globals.yml courtesy of chef-secrets.